.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
 * @actions/artifacts-actions
```

Comment:
Can we remove this group?  This means that `@actions/artifacts-actions` will get pinged on every PR.

You can still pull them in manually if you need support, but I think we should make it clear that your team owns this now.

Reply:
done